gcloud-java-examples/src/main/java/com/google/cloud/examples/compute/ComputeExample.java

```diff
@@ -83,9 +83,14 @@
  *
  * <p>This example demonstrates a simple/typical Compute usage.
  *
- * <p>See the README for the steps needed for compiling and running the example.
- * <p>Possible command-line arguments are:
- * <pre>{@code [<project_id>]
+ * <p>Steps needed for running the example:
+ * <ol>
+ * <li>log in using gcloud SDK - {@code gcloud auth login}.</li>
+ * <li>compile using maven - {@code cd gcloud-java-examples;
```

Comment:
The [README](https://github.com/GoogleCloudPlatform/gcloud-java/blob/gcs-nio/gcloud-java-examples/README.md) specifies that you first have to call `mvn install -DskipTests`. We should add this step here.

Though personally I prefer the [DRY](http://c2.com/cgi/wiki?DontRepeatYourself) approach of pointing the user to the README so we don't have multiple copies of the documentation and risk having them inconsistent.